libraries/AP_IOMCU/iofirmware/ioprotocol.h

```diff
@@ -58,6 +58,11 @@ enum iopage {
     PAGE_MIXING = 200,
     PAGE_GPIO = 201,
     PAGE_DSHOT = 202,
+    PAGE_RAW_DSHOT_ERPM = 203,
+    PAGE_RAW_DSHOT_TELEM_1_4 = 204,
+    PAGE_RAW_DSHOT_TELEM_5_8 = 205,
+    PAGE_RAW_DSHOT_TELEM_9_12 = 206,
+    PAGE_RAW_DSHOT_TELEM_13_16 = 207,
```

Comment:
why do we have pages for non-existant IOMCU groups?

Reply:
It was because we were defining IOMCU_MAX_CHANNELS as 16, I have now changed this to 8 and added an RC channels define